### ISSUE TITLE
Disabling microbreaks does not stop the next one

### ISSUE THREAD

**Issue by Quitch** [hovancik/stretchly#503]
### Prerequisites

- [X ] I'm using latest version: v0.21.0
- [X ] I've checked [Known issues](https://github.com/hovancik/stretchly#known-issues)

### Description

When disabling microbreaks in settings, if a microbreak is the next scheduled break type, the schedule is not updated until that microbreak has occurred.

Operating system: Windows 10 Pro x64 1909

### Steps to Reproduce

1. Ensure a microbreak is scheduled to occur next
2. Disable microbreaks

**Expected behavior:** The next break will be a break

**Actual behavior:** The scheduled microbreak will occur

**Reproduces how often:** 100%

### Additional Information

Resetting breaks after making the change correctly updates the schedule

**Comment by hovancik** [comment]
Hi, you are right. It is like that with more settings, ie when you change time between breaks it is changed only after next break. My thinking was that I don't wanna reset breaks everytime someone changes the settings, but what you say makes sense, it should be changes right away.

**Comment by Quitch** [comment]
Ideal behaviour would be that the break timings are unaffected and microbreaks simply removed from the schedule, but an automatic reset would work too.

**Comment by hovancik** [comment]
closing in favor of #1248
